packages/x-date-pickers/src/PickersActionBar/PickersActionBar.test.tsx

```diff
@@ -28,24 +28,6 @@ describe('<PickersActionBar />', () => {
     expect(screen.queryByRole('button')).to.equal(null);
   });
 
-  it('should not render buttons if actions function returns an empty array', () => {
```

Comment:
It is now `useSlotProps` that is responsible for having the same behavior with the callback and the object formats.